mmv1/templates/terraform/examples/dataplex_glossary_basic.tf.tmpl

```diff
@@ -0,0 +1,10 @@
+resource "google_dataplex_glossary" "{{$.PrimaryResourceId}}" {
+  provider = google-beta
+  glossary_id = "{{index $.Vars "glossary_name"}}"
+  location = "us-central1"
+  project      = "{{index $.TestEnvVars "project_name"}}"
```

Comment:
This isn't necessary here since the resource's project field will already default to the test project.

```suggestion
```